examples/client/getAccountInfo.ts

```diff
@@ -51,17 +51,19 @@ if (cli.flags.h) {
     cli.showHelp();
 }
 
-/// Retrieves information about an account. The function must be provided an account address or a credential registration id.
-/// If a credential registration id is provided, then the node returns the information of the account,
-/// which the corresponding credential is (or was) deployed to.
-/// If there is no account that matches the address or credential id at the provided
-/// block, then undefined will be returned.
+/// Retrieves information about an account. The function must be provided an
+/// account address or a credential registration id.  If a credential registration
```

Comment:
The node also supports querying by account index. Does this one not support it?

Reply:
Yes, the documentation here just doesn't mention it. I'll update it.